plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/procedure/MigrateProcedure.java

```diff
@@ -146,12 +152,15 @@ public MigrateProcedure(
             TrinoFileSystemFactory fileSystemFactory,
             TypeManager typeManager,
             IcebergConfig icebergConfig,
-            IcebergSecurityConfig securityConfig)
+            IcebergSecurityConfig securityConfig,
+            ExecutorService coreExecutor)
```

Comment:
nit: Move to after `TypeManager typeManager` to use the same order in the fields and reduce the diff.